admin/create-dumps.sh

```diff
@@ -63,6 +63,9 @@ function on_exit {
         local duration=$(( $(date +%s) - START_TIME ))
         echo "create-dumps took ${duration}s to run"
     fi
+
+    # Remove the cron lock
+    admin/cron_lock.py unlock-cron
```

Comment:
In all other places in this file where we call python scripts, we call `python scriptname.py` rather than using +x. In order to maintain consistency I'd prefer to explicitly run python here.

Reply:
Good call!